content/realtime/authentication.textile

```diff
@@ -117,15 +121,27 @@ Below is a rather contrived yet straightforward example that instances a Realtim
+
+h4(#ably-token). Ably Token
+
+An Ably Token is obtained through a "request":#request-token to Ably. This can either be done by the server, which then passes on the resulting Ably Token to the respective client, or by the client making use of an "Ably TokenRequest":#request-token. This token can also be embedded into an "External JWT":https://jwt.io as the @x-ably-token@ claim in the JWT's payload.
+
+h4(#ably-jwt). Ably JWT (JSON Web Token)
+
+"JWT (JSON Web Token)":https://jwt.io/ is an open standard (RFC 7519) that allows for compact secure information as JSON objects. Ably supports a JWT when the JWT is signed with an Ably API key. This type of JWT is called an "Ably JWT":/general/authentication#ably-jwt. A server can generate a JWT, and pass it to a client to be used. Unlike an Ably TokenRequest, it can be used as an actual token by the client, and the server can generate this token without the Ably library. Any third party software can be used to generate this JWT using the Ably private API key to sign it.
+
 h3(#auth-object). Auth object
 
-The principal use-case for @Auth@ object is to "create signed token request objects":#create-token-request or "obtain tokens from Ably":#request-token, and then issue them to other "less trusted" clients. Typically, your servers should be the only devices to have a "private API key":https://support.ably.io/solution/articles/3000030054-what-is-an-app-api-key, and this private API key is used to securely sign token requests or request tokens from Ably. Clients are then issued with short-lived tokens or token requests, and the libraries can then use these to authenticate with Ably. If you adopt this model, your private API key is never shared with clients directly.
+The principal use-case for @Auth@ object is to "create Ably TokenRequest objects":#create-token-request or "obtain Ably Tokens from Ably":#request-token, and then issue them to other "less trusted" clients. Typically, your servers should be the only devices to have a "private API key":https://support.ably.io/solution/articles/3000030054-what-is-an-app-api-key, and this private API key is used to securely sign Ably TokenRequests or request Ably Tokens from Ably. Clients are then issued with short-lived Ably Tokens or Ably TokenRequests, and the libraries can then use these to authenticate with Ably. If you adopt this model, your private API key is never shared with clients directly.
 
-A subsidiary use-case for the @Auth@ object is to preemptively trigger renewal of a token or to acquire a new token with a revised set of capabilities by explicitly calling "<span lang="default">@authorize@</span><span lang="csharp">@Authorize@</span>":#authorize.
+A subsidiary use-case for the @Auth@ object is to preemptively trigger renewal of an Ably Token or to acquire a new Ably Token with a revised set of capabilities by explicitly calling "<span lang="default">@authorize@</span><span lang="csharp">@Authorize@</span>":#authorize.
```

Comment:
Token renewal can apply to any token type. Again, I would just remove the "Ably" qualiifer.